src/controllers/UsersController.php

```diff
@@ -298,4 +298,38 @@ public function passwordReset(Request $request, $db)
         }
 
     }
+
```

Comment:
Well, my house is very small, so putting extra space into code makes me feel better :-p

Will edit and push